sdk/program/src/lib.rs

```diff
@@ -179,6 +180,17 @@ macro_rules! unchecked_div_by_const {
     }};
 }
 
+#[derive(Debug, Error)]
```

Comment:
This seems out of place here, maybe put in `lamporrts.rs`?  also the future home of a lamports type ;-)

Reply:
moved